lms/djangoapps/ci_lrs/tests.py

```diff
@@ -0,0 +1,50 @@
+import random
+from unittest.mock import patch, MagicMock
+
+from django.conf import settings
+from django.utils import timezone
+from django.test import TestCase, override_settings
+
+from pymongo import MongoClient
+from pymongo.errors import ConnectionFailure, DuplicateKeyError, NetworkTimeout, OperationFailure, ServerSelectionTimeoutError
+
+from ci_lrs.utils import write_lrs_record_to_mongo
+
+
+class TestLRSTasks(TestCase):
+    def setUp(self):
+        self.mongodb = MongoClient('mongodb://mongodb:27017')['test_db']
+        self.mongodb['student_activity'].delete_many({})
+
+    def test_write_lrs_record_to_mongo(self):
+        context = {
+            'activity_time': timezone.now().isoformat(),
```

Comment:
The activity time in the edx webhook lambda is saved as a datetime object rather than a ISO string. Might be good to keep doing that.

Reply:
The string is parsed back to a date in the theme since you cannot pass a DateTime object to a task.